src/main/java/cz/fi/muni/pa165/esports/dao/MatchRecordDao.java

```diff
@@ -2,19 +2,54 @@
 
 
 import cz.fi.muni.pa165.esports.entity.MatchRecord;
+import cz.fi.muni.pa165.esports.entity.Player;
 
 import java.util.List;
 
-/*
-    @Author Elena Álvarez
-*/
+/**
+ * A DAO interface for the MatchRecord entity.
+ *
+ * @author Elena Álvarez
+ */
 public interface MatchRecordDao {
-    public MatchRecord findById(Long id);
+    /**
+     * Adds a MatchRecord to the database.
+     * @param c entity to be persisted
+     */
     public void create(MatchRecord c);
+
+    /**
+     * Deletes a MatchRecord from the database.
+     * @param c entity to be deleted
+     */
     public void delete(MatchRecord c);
+
+    /**
+     * Fetches a MatchRecord according to its ID from the database.
+     * @param id of the sought team entity
+     * @return MatchRecord entity if found, else null
+     */
+    public MatchRecord findById(Long id);
+
+    /**
+     * Fetches all MatchRecords.
+     * @return list of all MatchRecord entities in the database
+     */
     public List<MatchRecord> findAll();
-    public MatchRecord findByName(String name);
-    public MatchRecord findByMatchNumber(Long match_number);
-    //public Match findByPlayer(Player player);
```

Comment:
I think the `findByPlayer` method could be very useful in the future, could you restore and implement that and add javadoc?

The method would return all `MatchRecords` for a given player, no need to limit results based on anything else, just the player. I think we'll need it for player statistics.

Reply:
Done it